importer/markdown/src/test/java/org/itsallcode/openfasttrace/importer/markdown/ITMarkdownMarkupImporter.java

```diff
@@ -0,0 +1,139 @@
+package org.itsallcode.openfasttrace.importer.markdown;
+
+import static org.itsallcode.matcher.auto.AutoMatcher.contains;
+import static org.itsallcode.openfasttrace.testutil.core.ItemBuilderFactory.item;
+import static org.itsallcode.openfasttrace.testutil.importer.ImportAssertions.assertImportWithFactory;
+
+import org.hamcrest.Matcher;
+import org.itsallcode.openfasttrace.api.core.SpecificationItem;
+import org.itsallcode.openfasttrace.api.core.SpecificationItemId;
+import org.itsallcode.openfasttrace.api.importer.ImporterFactory;
+import org.itsallcode.openfasttrace.importer.lightweightmarkup.AbstractLightWeightMarkupImporterTest;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+class ITMarkdownMarkupImporter extends AbstractLightWeightMarkupImporterTest {
```

Comment:
Could we rename this to `TestMarkdownMarkupImporter` so that it runs with normal unit tests? It should be quick enough as it doesn't touch files or network.